### PR TITLE
Civ 5572 dispute paid less

### PR DESCRIPTION
**Pull request by neeta-hmcts**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-5572

### Change description ###
Defendant response view for "Dispute + paid what I owed" scenario.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/common/models/claim.ts [hunk 1]
```diff
@@ -264,6 +283,18 @@ export class Claim {
     return this.rejectAllOfClaim.howMuchHaveYouPaid.amount === this.rejectAllOfClaim.howMuchHaveYouPaid.totalClaimAmount;
   }
```
Comment (neeta-hmcts) [inline src/main/common/models/claim.ts:285]: CIV-5572 changes

File: src/main/services/features/claimantResponse/defendantResponse/defendantResponseSummaryService.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+} from './fullDisputeDefendantsResponseContent';
+import {buildFullAdmissionResponseContent} from './fullAdmissinionDefendantsResponseContent';
+
+export const getDefendantsResponseContent = (claim: Claim, lang: string): ClaimSummarySection[] => {
+  switch (claim.responseStatus) {
+    case ClaimResponseStatus.FA_PAY_BY_DATE:
+      return buildFullAdmissionResponseContent(claim, lang);
+    case ClaimResponseStatus.RC_DISPUTE:
+      return buildFullDisputeResponseContent(claim, lang);
+    case ClaimResponseStatus.RC_PAID_LESS:
```
Comment (neeta-hmcts) [inline src/main/services/features/claimantResponse/defendantResponse/defendantResponseSummaryService.ts:16]: CIV-5572 changes

File: src/main/services/features/claimantResponse/defendantResponse/fullDisputeDefendantsResponseContent.ts [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+  return [
+    ...getResponseStatement(claim.getDefendantName(), 'PAGES.REVIEW_DEFENDANTS_RESPONSE.REJECT_CLAIM_STATEMENT'),
+    ...getTheirDefence(claim.rejectAllOfClaim?.defence?.text),
+    ...getTheirTOEs(claim, lng),
+    ...getDisagreementStatementWithTimeline(claim),
+    ...getTheirEvidence(claim, lng),
+    ...getDisagreementStatementWithEvidence(claim),
+  ];
+};
+export const buildFullDisputePaidLessResponseContent = (claim: Claim, lng: string): ClaimSummarySection[] => {
```
Comment (neeta-hmcts) [inline src/main/services/features/claimantResponse/defendantResponse/fullDisputeDefendantsResponseContent.ts:220]: CIV-5572 changes

File: src/test/unit/services/features/claimantResponse/defendantResponse/defendantResponseSummaryService.test.ts [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+    expect(defendantsResponseContent[8].data?.text).toEqual('PAGES.REVIEW_DEFENDANTS_RESPONSE.THEIR_EVIDENCE');
+    expect(defendantsResponseContent[9].data?.head[0].text).toEqual('COMMON.EVIDENCE_SUMMARY.ROW_TYPE');
+    expect(defendantsResponseContent[9].data?.head[1].text).toEqual('COMMON.DESCRIPTION');
+    expect(defendantsResponseContent[9].data?.tableRows[0][0].text).toEqual('Contracts and agreements');
+    expect(defendantsResponseContent[9].data?.tableRows[0][1].text).toEqual('I have a signed contract showing that you broke the contract agreement.');
+    expect(defendantsResponseContent[10].data?.text).toEqual('PAGES.REVIEW_DEFENDANTS_RESPONSE.WHY_THEY_DISAGREE_EVIDENCE');
+    expect(defendantsResponseContent[11].data?.text).toEqual('evidence comments');
+  });
+
+  describe('Full dispute Paid less scenario', () => {
```
Comment (neeta-hmcts) [inline src/test/unit/services/features/claimantResponse/defendantResponse/defendantResponseSummaryService.test.ts:89]: CIV-5572 changes

File: src/main/services/features/claimantResponse/defendantResponse/defendantResponseSummaryService.ts [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import {Claim} from 'models/claim';
```
Comment (kenneth-hmcts) [inline src/main/services/features/claimantResponse/defendantResponse/defendantResponseSummaryService.ts:1]: is it better using something like for the import ../../models/claim

File: src/main/routes/features/claimantResponse/reviewDefendantsResponseController.ts [hunk 6]
```diff
@@ -0,0 +1,36 @@
+import {NextFunction, Request, Response, Router} from 'express';
```
Comment (kenneth-hmcts) [inline src/main/routes/features/claimantResponse/reviewDefendantsResponseController.ts:1]: is it better using ../../ for the import

**Review by laperla11 — APPROVED** [review]

File: src/main/routes/features/claimantResponse/reviewDefendantsResponseController.ts (hunk 6 above)
Comment (lfazelyniaHMCTS) [reply src/main/routes/features/claimantResponse/reviewDefendantsResponseController.ts:1]: I think this is ok as just 'express' as it looks to be consistent with all other controllers

Comment (kenneth-hmcts) [reply src/main/routes/features/claimantResponse/reviewDefendantsResponseController.ts:1]: Sorry that I should put it on the next line after the import express

Comment (neeta-hmcts) [reply src/main/routes/features/claimantResponse/reviewDefendantsResponseController.ts:1]: ok.. I will update the import paths but I have worked on this TD ticket few days back CIV-1350-TechDebt so I thought it is ok to use an absolute path to fix the import warning.

File: src/main/services/features/claimantResponse/defendantResponse/defendantResponseSummaryService.ts (hunk 5 above)
Comment (kenneth-hmcts) [reply src/main/services/features/claimantResponse/defendantResponse/defendantResponseSummaryService.ts:1]: Ok, is it same for the controller part?

**Review by kenneth-hmcts — APPROVED** [review]

File: src/main/services/features/claimantResponse/defendantResponse/defendantResponseSummaryService.ts (hunk 5 above)
Comment (commitBlob) [reply src/main/services/features/claimantResponse/defendantResponse/defendantResponseSummaryService.ts:1]: Yep applies for the whole project

**Pull request merged**